test/e2e/tests/e2e_test.go

```diff
@@ -147,10 +152,62 @@ func test(t *testing.T, testFunc testFunc) {
 			t.Skip("Skipping AVS tests since no deploy info is available")
 			return
 		}
-		// TODO: Create AWS contracts and test them
-		for chain, info := range deployInfo {
-			t.Log(chain, info)
+
```

Comment:
let's use the same pattern as we do above:
- keep this function simple and readable
- add a `makeAVS` function that configures addresses